src/cleartext.js

```diff
@@ -111,9 +111,9 @@ export class CleartextMessage {
    * @returns {String | ReadableStream<String>} ASCII armor.
    */
   armor(config = defaultConfig) {
+    const includesNonV6Signatures = this.signature.packets.some(packet => packet.version !== 6);
     // emit header if one of the signatures has a version not 6
-    const emitHeader = this.signature.packets.some(packet => packet.version !== 6);
-    const hash = emitHeader ?
+    const hash = includesNonV6Signatures ?
```

Comment:
Bit nitpicky but I think it's easier to understand the code if this variable says what it's used for rather than how it was computed, closer to how it was before:
```js
    // Emit header and checksum if one of the signatures has a version other than 6
    const emitHeaderAndChecksum = this.signature.packets.some(packet => packet.version !== 6);
    const hash = emitHeaderAndChecksum ?
```